### ISSUE TITLE
Cannot Save Photo after Editing

### ISSUE THREAD

**Issue by mariobehling** [fossasia/phimpme-android#2771]
User Report: 

I like app appreciate you writing it , but on my Motorola phone G5 plus S , there is no workable option to save the photo after having edited it. when I am in the editing screen , and click the check mark it goes back to the editing screen . When I click the top right icon , nothing happens. when I back out of the app , it says I am leaving without saving and offers to save . I can click to save , as far as I can tell nothing at all happens. If I try to back out again after that it brings up the same option to save or discard dialogue. I can't find information on this on the wiki page .
 Thanks for any thoughts you have

**Comment by yashk2000** [comment]
@mariobehling this isn't a valid issue in the current version of the app. I tested it properly. 
To save, after editing, click the tick button, then you'll be back on the main editing screen. Here, on the top right of the edit bar, there is an exit button(a rectangle with an arrow on it). If you press it, the edited image gets saved.
Should I close this issue?

**Comment by yashk2000** [comment]
@iamareebjamal @abishekvashok @mariobehling should I close this issue?

**Comment by Mindstormer619** [comment]
I have the same issue on the release version of the app that I downloaded just about now. Is the issue fixed only in the development version?

**Comment by yashk2000** [comment]
@Mindstormer619 that might be a permissions issue. Try going to the app permission settings and manually toggling the storage permission. I have tried the app on several phones, but have never faced this issue.

**Comment by yashk2000** [comment]
@iamareebjamal @abishekvashok @mariobehling should this issue be closed?

**Comment by iamareebjamal** [comment]
2 people have reported the same issue

**Comment by yashk2000** [comment]
@iamareebjamal @mariobehling I was able to find a phone on which I faced this issue. The storage permission was already given by the user but still, this issue happened. 
When I manually went to the permission setting of the app and toggled the storage permission off and then on again, the app worked properly as expected. I was not able to find any reason as to why this happened. 
So in case, this happens, should I display a dialog to the user and direct them to permissions setting with the message that they need to manually toggle the permissions?
